README.md

```diff
@@ -1,92 +1,111 @@
 
 - [Terraform](https://www.terraform.io/intro/index.html)
 - [Docker](https://www.docker.com/)
 - [Microsoft Azure Cloud](https://azure.microsoft.com/)
 - [Kubernetes](https://kubernetes.io/docs/concepts/)
  
 
-### Technical Prerequisites:
+### Technical Prerequisites
+
+This project supports two options for running Terraform scripts:
+- Terraform installed on your local machine
+- Using a Docker container to run Terraform
+  
+  For more information, see [Docker Usage](./docs/user/DockerUsage.md).
 
-This project supports running **either** via terraform installed on your local machine **or** via a docker container. 
+Access to an **Azure Subscription** and an [**Identity**](./docs/user/TerraformAzureAuthentication.md) with the *Contributor* role are required.
 
-- Access to an **Azure Subscription** and [**Identity**](./docs/user/TerraformAzureAuthentication.md) with '*Contributor*' role
-  - #### Terraform
-    - [Terraform](https://www.terraform.io/downloads.html) - v1.0.0
-    - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) - v1.19.9
-    - [jq](https://stedolan.github.io/jq/) - v1.6
-    - [Azure CLI](https://docs.microsoft.com/en-us/cli/azure) - (optional - useful as an alternative to the Azure Portal) - v2.24.2
-  - #### Docker
-    - [Docker](https://docs.docker.com/get-docker/)
+#### Terraform Requirements:
+- [Terraform](https://www.terraform.io/downloads.html) - v1.0.0
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) - v1.19.9
+- [jq](https://stedolan.github.io/jq/) - v1.6
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure) - (optional - useful as an alternative to the Azure Portal) - v2.24.2
 
+#### Docker Requirements:
+- [Docker](https://docs.docker.com/get-docker/)
 
 ## Getting Started
 
-### Clone this project
+When you have prepared your environment with the prerequisites, you are ready to obtain and customize the Terraform scripts that
+will set up your Kubernetes cluster.
 
-Run these commands in a Terminal session:
+### Clone this Project
+
+Run the following commands from a terminal session:
 
 ```bash
 # clone this repo
 git clone https://github.com/sassoftware/viya4-iac-azure
 
-# move to directory
+# move to the project directory
 cd viya4-iac-azure
 ```
 
-### Authenticating Terraform to access Azure
+### Authenticating Terraform to Access Microsoft Azure
+
+The Terraform process manages Microsoft Azure resources on your behalf. In order to do so, it needs your Azure account information and a user
+identity with the required permissions. See [Terraform Azure Authentication](./docs/user/TerraformAzureAuthentication.md) for details.
 
-See [Terraform Azure Authentication](./docs/user/TerraformAzureAuthentication.md) for details.
+### Customizing Input Values
 
-### Customize Input Values
+Terraform scripts require variable definitions as input. Many variables do not have default values. Create a file named
```

Comment:
The statement Many variables do not have default values is a bit misleading. If the user chooses to use the `default` sample file which has very few lines, a cluster will be created. Most exposed variables have defaults which allows very minimal effort by the end user when creating a cluster.  Either remove the sentence or reword. Not sure what would go in its place.